src/ClassicUO.Client/LegionScripting/docs/API.py

```
@@ -15,6 +15,9 @@ def Add(self, childControl: PyControl) -> None:
         """
         pass
 
+    def Add(self, childControl: Any) -> None:
```

Ruff (0.13.1)

Redefinition of unused `Add` from line 10

(F811)